packages/playground/src/components/contracts_list/contracts_table.vue

```diff
@@ -89,6 +89,41 @@
     </template>
   </weblet-layout>
 
+  <v-dialog width="800" v-model="contractStateDialog">
+    <v-card>
+      <v-card-title class="text-h5 mt-2"> Contract lock Details</v-card-title>
+      <v-card-text>
+        <v-row class="d-flex justify-center">
+          Amount Locked:
+          {{ contractLocked?.amountLocked ?? 0 > 0 ? contractLocked?.amountLocked.toFixed(3) : 0 }}
```

Comment:
Please create a function that does this logic and add only the variable/function caller in the template

```suggestion
          {{ getAmountLocked() }}
          
  const getAmountLocked = (): number => {
    const amountLocked = contractLocked?.amountLocked ?? 0;
    return amountLocked > 0 ? parseFloat(amountLocked.toFixed(3)) : 0;
  };
```